recipes/list_items/contact_info/contact_info.vue

```diff
@@ -219,3 +220,25 @@ export default {
   },
 };
 </script>
+
+<style>
```

Comment:
Done, please review.

Reply:
Hi @iropolo , sorry, I just find this after I merge it. With `scoped` tag, the min-width override does not work. Is that because when `scoped` tag, it can only override `contact_info` (and if vue find this is not `contact_info` component element, it will just ignore it)?  I will make another pull request to fix this.